crates/python/qcs_sdk/__init__.pyi

```diff
@@ -306,28 +316,27 @@ class ResultData:
         ...
     def to_register_map(self) -> RegisterMap:
         """
-        Convert ``ResultData`` from its inner representation as ``QVMResultData`` or
-        ``QPUResultData`` into a ``RegisterMap``. The ``RegisterMatrix`` for each register will be
+        Convert `ResultData` from its inner representation as `QVMResultData` or
```

Comment:
If we're going from double to single backtick, will this still render correctly in IDEs?

Reply:
LSP supports GitHub flavored markdown which also renders ``inline code`` between double backticks. I installed VSCode to double check.

<img width="849" alt="image" src="https://github.com/rigetti/qcs-sdk-rust/assets/4324359/caf14b69-89e5-4838-9498-2873b09a5177">